src/server/node_manager.c

```diff
@@ -7261,6 +7360,12 @@ update_job_node_rassn(job *pjob, attribute *pexech, enum batch_op op)
 			return;
 	}
 
+	if (pjob && (pjob->ji_qs.ji_svrflags & JOB_SVFLG_Broadcast_Rqd)) {
+		psvr_ru = init_ru(pjob, op, pexech->at_val.at_str);
+		mcast_resc_usage(psvr_ru);
```

Comment:
Instead of sending resource update to all peer servers why can't we send it to the servers which need them ?